addons/remote-write-ingress.libsonnet

```diff
@@ -0,0 +1,93 @@
+function(config) {
+  assert std.objectHas(config, 'prometheus') : 'prometheus required',
+  assert std.objectHas(config.prometheus, 'DNS') : 'prometheus.DNS required',
+  assert std.objectHas(config.prometheus, 'nodePort') : 'prometheus.nodePort required',
+  assert std.isNumber(config.prometheus.nodePort) : 'prometheus.nodePort should be a number',
+  assert std.objectHas(config.prometheus, 'GCPExternalIpAddress') : 'prometheus.GCPExternalIpAddress required',
+  assert std.objectHas(config.prometheus, 'BasicAuthSecret') : 'prometheus.BasicAuthPassword required',
+
+
+  prometheus+: {
+    certificate: {
+      apiVersion: 'cert-manager.io/v1',
+      kind: 'Certificate',
+      metadata: {
+        name: 'prometheus-remote-write',
+        namespace: config.namespace,
+      },
+      spec: {
+        dnsNames: [
+          config.prometheus.DNS,
+        ],
+        issuerRef: {
+          kind: 'ClusterIssuer',
+          name: 'letsencrypt-issuer',
```

Comment:
Hmmmm, If I'm not mistaken this issuer doesn't exist anymore 🤔...

This is the old issuer that was creating certificates under the old google project `gitpod-dev`, let me check what is the one used on `gitpod-core-dev`

Reply:
It should be the one that has access to the google project `gitpod-core-dev`

Reply:
```
werft                          werft-grpc-ingress-certificate         True    werft-grpc-ingress-certificate         letsencrypt-issuer                   Certificate is up to date and has not expired   21d
werft                          werft-ingress-certificate              True    werft-ingress-certificate              letsencrypt-issuer                   Certificate is up to date and has not expired   302d
```
Seems like werft uses this one